client/web/src/settings/DynamicallyImportedMonacoSettingsEditor.tsx

```diff
@@ -192,7 +192,7 @@ export class DynamicallyImportedMonacoSettingsEditor<T extends object = {}> exte
 
     private monacoRef = (monacoValue: typeof _monaco | null): void => {
         this.monaco = monacoValue
-        if (this.monaco && MonacoSettingsEditor._result) {
+        if (this.monaco) {
```

Comment:
The original before the change was 

```suggestion
        if (this.monaco && MonacoSettingsEditor) { 
```

does it matter?

Reply:
I really can't understand the original reason for having this check with just `MonacoSettingsEditor`, because it always is equal to `LazyExoticComponent` which will be resolved as true.

Reply:
I actually did some code archaeology before opening this PR to figure out where that came from, and I think `MonacoSettingsEditor` may originally have been a component, rather than a function, and the `if` never got fully rethought when it changed. I'm comfortable dropping the check.